go.mod

```diff
@@ -16,7 +16,7 @@ require (
 	github.com/imdario/mergo v0.3.7
 	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822
 	github.com/opencontainers/go-digest v1.0.0
-	github.com/openshift/api v0.0.0-20230613151523-ba04973d3ed1
```

Comment:
As mentioned in slack, the openshift/api module is replaced so you'll need to something similar to what was mentioned in https://github.com/openshift/cluster-ingress-operator/pull/900#pullrequestreview-1478216617